src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -365,16 +369,16 @@ public static void setGlobalNIODefaultOptions() {
         final String[] split = gcsUrl.split("/", -1);
         final String BUCKET = split[2];
         final String pathWithoutBucket = String.join("/", Arrays.copyOfRange(split, 3, split.length));
-        CloudStorageConfiguration cloudConfig = getCloudStorageConfiguration();
+        CloudStorageConfiguration cloudConfig = getCloudStorageConfiguration(DEFAULT_NIO_MAX_REOPENS);
```

Comment:
This call will only use the config with DEFAULT_NIO_MAX_REOPENS rather than what is set on the command-line.  You may need to pass in the new nioMaxReopens to this method.